### PR TITLE
Update dataset_infos for UDHN/udhr dataset

### PR DESCRIPTION
**Pull request by leondz**
Checksum update to `udhr` for issue #4361

### CODE REVIEW

**Comment by albertvillanova** [comment]
Thanks for contributing @leondz.

The checksums of the files have changed because more languages have been added:
- the new language codes need to be added to the dataset card (README file)
- I think the dataset version number should also be increased, so that users who had previously cached it, get a new dataset download (with the additional languages)

**Comment by leondz** [comment]
Yep! All done

**Review by albertvillanova — CHANGES_REQUESTED** [review]

File: datasets/udhr/udhr.py [hunk 1]
```diff
@@ -46,6 +46,8 @@
 class UDHN(datasets.GeneratorBasedBuilder):
     """Universal Declaration of Human Rights"""
 
+    VERSION = datasets.Version("1.2022.0519")
```
Comment (albertvillanova) [inline datasets/udhr/udhr.py:49]: Also note that we normally use semantic versioning.

Comment (leondz) [reply datasets/udhr/udhr.py:49]: Do you have a link to an acceptable scheme?

File: datasets/udhr/README.md [hunk 2]
```diff
@@ -8,73 +8,102 @@ languages:
...
 - am
 - amc
 - ame
 - ami
 - amr
-- arb
+- ar
 - arl
 - arn
+- as
```
[the remark this answers is not in the document]
Comment (albertvillanova) [reply datasets/udhr/README.md:32]: Normally, we would like to be in sync with the data provided by the data owner.
- If we use their source URL, our checksum will raise an exception each time the data owner updates their data; that way we will know we need to update our version (and add the new languages to the dataset card)
- If we make a dump of current data, we will not notice there are updates unless we check periodically their website

I think it is better to have an exception that tells us to make an update on our side. Also note that users can circumvent this exception if they really want to load the dataset as it is, by passing `ignore_verifications=True`

File: datasets/udhr/udhr.py (hunk 1 above)
Comment (leondz) [reply datasets/udhr/udhr.py]: This is the first version with a version number, so bumped to 1.0.0

File: datasets/udhr/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (leondz) [reply datasets/udhr/README.md]: OK, cool. Fixed now (stage 1+2 removed)

If anyone looks at this later, there's a repo for the data at https://github.com/unicode-org/udhr which seems to maintain a dated revision history of the source data, and that should allow post-hoc determination of which HF versions corresponded to what translations.

**Comment by leondz** [comment]
I guess the language code CI failure is due to languages.json being a subset of bcp47 (see issue #4304), happy to contribute a solution here, e.g. autogeneration of the lang list from the relevant isos and the ietf bcp47 subtag register or full code for validation

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Thanks again for your contribution.

Yes, I think it is OK to set version 1.0.0 (as previous was 0.0.0).

One of the CI failures is related to dummy data: once you have updated the dataset version, the dummy_data ZIP file should be moved from "dummy/0.0.0/dummy_data.zip" to "dummy/1.0.0/dummy_data.zip".

Other CI failure is related to missing languages in our resources file. This has been addressed in this PR:
- #4371 

You should merge master branch into your feature branch to incorporate that fix.

**Comment by leondz** [comment]
> Thanks again for your contribution, @leondz.
> 
> Yes, I think it is OK to set version 1.0.0 (as previous was 0.0.0).
> 
> One of the CI failures is related to dummy data: once you have updated the dataset version, the dummy_data ZIP file should be moved from "dummy/0.0.0/dummy_data.zip" to "dummy/1.0.0/dummy_data.zip".

Oh, thanks, I missed that one


> Other CI failure is related to missing languages in our resources file. This has been addressed in this PR:
> 
> * #4371
> 
> You should merge master branch into your feature branch to incorporate that fix.

Yeah, I saw this :) I already have the merge, thanks. I'm talking about the longer-term picture: every time another language code comes up (e.g. da-bornholm or es-VE), the json will need updating, because the current approach is non-exhaustive manual whitelisting instead of relying on the established bcp standard.

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Hi @leondz,

Could you please merge the master branch to see if the tests pass?
```
git fetch upstream master
git merge upstream/master
git push
```

**Comment by albertvillanova** [comment]
@leondz you need to merge master branch again to fix the tests.

**Review by albertvillanova — APPROVED** [review]
Thank you!

**Pull request merged**